modules/functions.js

```diff
@@ -0,0 +1,130 @@
+  const deleteBtn = document.createElement('span');
+
+  // Adding class
+  taskDiv.className = 'task';
+  deleteBtn.className = 'removeBtn';
+  Input.className = 'task-value';
+  taskStatus.className = 'checkTask';
+
+  // Setting Attribute
+  taskStatus.setAttribute('type', 'checkbox');
+
+  // Display Elements
+  taskStatus.checked = todos.completed;
+  Input.value = todos.description;
+  ellipsis.innerHTML = '<i class="fa fa-ellipsis-h" aria-hidden="true"></i>';
+  deleteBtn.innerHTML = '<i class="fa fa-trash" aria-hidden="true"></i>';
+
+  // Appending element
+  taskDiv.appendChild(taskStatus);
+  taskDiv.appendChild(Input);
+  taskDiv.appendChild(ellipsis);
+  taskDiv.appendChild(deleteBtn);
+  tasks.appendChild(taskDiv);
+}
+
+// function to get task from local storage and display from storage
+// eslint-disable-next-line no-unused-vars
+const getTask = () => {
+  todoList = JSON.parse(localStorage.getItem('tasks: '));
+  if (todoList !== null) {
+    todoList.forEach((todos) => {
+      taskDisplay(todos);
+    });
+  } else {
+    todoList = [];
+  }
+};
+
+// Creating the todo object structure
+// eslint-disable-next-line no-unused-vars
+const taskObjects = (todos) => {
+  const task = {
+    description: String,
+    completed: false,
+    index: Number,
+  };
+  task.description = todos;
+  task.index = todoList.length;
+  addToLocaleStorage(task);
+  taskDisplay(task);
+};
+
+// function to delete a task from the list
+// eslint-disable-next-line no-unused-vars
+const removeTask = (index) => {
+  const tasksElement = document.querySelectorAll('.task');
+  todoList.splice(index, 1);
+  tasksElement[index].remove();
+  for (let i = index + 1; i < todoList.length; i += 1) {
+    todoList[i] -= 1;
+  }
+  localStorage.setItem('tasks: ', JSON.stringify(todoList));
+};
+
+// function to modify each task
+// eslint-disable-next-line no-unused-vars
+const editTask = (todos, index) => {
+  todoList[index].description = todos;
+  localStorage.setItem('tasks: ', JSON.stringify(todoList));
+};
+
+const highlightTask = (index) => {
+  const moveBtns = document.querySelectorAll('.fa.fa-ellipsis-h');
+  const deleteBtn = document.querySelectorAll('.fa.fa-trash');
+  const activeTasks = document.querySelectorAll('.task.active');
+  const tasksElt = document.querySelectorAll('.task');
+  activeTasks.forEach((activeTask) => {
+    activeTask.classList.remove('active');
+  });
+  moveBtns.forEach((btn, index) => {
+    btn.classList.remove('active');
+    deleteBtn[index].classList.remove('active');
+  });
+
+  tasksElt[index].classList.toggle('active');
+  moveBtns[index].classList.toggle('active');
+  deleteBtn[index].classList.toggle('active');
+};
+
+// function for the clear completed button, hint: its not functional yet
+const clearCompleted = (index) => {
+  const checkedTask = document.querySelectorAll('.checkedTask');
+  checkedTask.forEach((task) => {
+    if (task.completed === true) {
+      removeTask(index);
+    }
+  });
+};
```

Comment:
- [ ] Clear the completed tasks are not working, kindly look into that https://www.loom.com/share/a11bf2ccb1e647438eb67e37b7d01dee

Reply:
I have fixed the changes if you Add, then click the checkbox and clicked clear completed it deletes